src/metrics-server/chart/templates/service-monitor.yaml

```diff
@@ -3,7 +3,7 @@
 apiVersion: monitoring.coreos.com/v1
 kind: ServiceMonitor
 metadata:
-  annotation:
+  annotations:
```

Comment:
Ty for fixing my problem ❤️ - for awareness to others this was the issue with metrics-server. Zarf/helm do not explode on extra fields in a manifest so this being incorrect was obscured.